py2-pyOpenSSL.spec

```diff
@@ -0,0 +1,4 @@
+### RPM external py2-pyOpenSSL 18.0.0
+## IMPORT build-with-pip
+
+Requires: py2-cryptography
```

Comment:
you missed a dependency on py2-six here

Reply:
py2-cryptography brings in that dependency
https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_10_4_X/gcc700/py2-cryptography.spec#L4